packages/block-library/src/post-author/edit.js

```diff
@@ -100,74 +101,82 @@ function PostAuthorEdit( {
 		<>
 			<InspectorControls>
 				<PanelBody title={ __( 'Settings' ) }>
-					{ showAuthorControl &&
-						( ( showCombobox && (
-							<ComboboxControl
-								__nextHasNoMarginBottom
-								label={ __( 'Author' ) }
-								options={ authorOptions }
-								value={ authorId }
-								onChange={ handleSelect }
-								allowReset={ false }
-							/>
-						) ) || (
+					<VStack
+						spacing={ 4 }
+						className="wp-block-post-author__inspector-settings"
+					>
+						{ showAuthorControl &&
+							( ( showCombobox && (
+								<ComboboxControl
+									__next40pxDefaultSize
```

Comment:
The spacing gets messed up when a ComboboxControl enters the mix, because it has an additional div wrapper outside the BaseControl. The auto margin added to BaseControls by the block inspector interprets this as a `:last-child` BaseControl, and adds a smaller margin.

So here we add a `VStack` to explicitly handle our spacing, with the correct metrics (16px gap), and additional CSS to counteract the auto margins. Painful 😭 FYI I am working on this in #64526.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/9a8a1528-0b08-4679-9287-9aeb8012343d" alt="Author block, before" width="275">|<img src="https://github.com/user-attachments/assets/e9680946-3f0b-4b5e-8d55-8dccab4c2654" alt="Author block, after" width="275">|

Looks like this when `showCombobox === false`:

<img src="https://github.com/user-attachments/assets/cc5ac6f6-9359-4840-adb2-d4a05f4fc37a" alt="Author block, after with select" width="275">

Reply:
> Painful

:hug: